lms/djangoapps/courseware/features/navigation.feature

```diff
@@ -18,4 +18,4 @@ Feature: LMS.Navigate Course
         Given I am viewing a section with multiple sequences
         When I navigate to an item in a sequence
         Then I see the content of the sequence item
-        And an "edx.ui.lms.sequence.tab_selected" browser event is emitted
+        And a "seq_goto" browser event is emitted
```

Comment:
Because the test checks the event_type, not the name field.
On Apr 21, 2016 11:52 AM, "Nimisha Asthagiri" <notifications@github.com>
wrote:

> In lms/djangoapps/courseware/features/navigation.feature
> <https://github.com/edx/edx-platform/pull/12127#discussion_r60606430>:
>
> > @@ -18,4 +18,4 @@ Feature: LMS.Navigate Course
> >          Given I am viewing a section with multiple sequences
> >          When I navigate to an item in a sequence
> >          Then I see the content of the sequence item
> > -        And an "edx.ui.lms.sequence.tab_selected" browser event is emitted
> > +        And a "seq_goto" browser event is emitted
>
> Why did this need to be reverted?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/12127/files/d60ab18593c17d496ceda2cb9d9b28babcee0d50..1b674853948096c7ef46a5ed678641357f995400#r60606430>
>